src/pfe/portal/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "server.js",
   "scripts": {
     "start": "./npm-start.sh",
-    "prod": "node server.js",
-    "dev": "nodemon server.js",
```

Comment:
Removing this also removed the dev environment on whole PFE. The dev environment was useful to easily push changes without rebuilding PFE. https://github.com/eclipse/codewind/issues/695